modules/custom/cu_faculty_publications_bundle/cu_faculty_publications_bundle.module

```diff
@@ -43,142 +43,261 @@ function cu_faculty_publications_bundle_theme(&$existing, $type, $theme, $path)
  * Implements hook_preprocess_entity().
  */
 function cu_faculty_publications_bundle_preprocess_entity(&$vars) {
-  // $entity_type = $vars['elements']['#entity_type'];
   if ($vars['elements']['#bundle'] == 'faculty_publications') {
 
-    $endpoint = variable_get('cu_faculty_publications_endpoint', 'https://experts.colorado.edu/es/fispubs-v1/publication/_search');
-    $fisEndpoint = variable_get('cu_faculty_fis_endpoint', 'https://experts.colorado.edu/es/fis/_search');
-    $arguments = array();
-    $numberOfResultsPerPage = 25;
-    $requestedNumberOfResults = 10;
-    $currentPage = pager_find_page();
-
-    // Check for publication date filter.
-    if (!empty($vars['bean']->field_faculty_publication_date)) {
-      $date1 = strtotime($vars['bean']->field_faculty_publication_date[LANGUAGE_NONE][0]['value']);
-      $date2 = strtotime($vars['bean']->field_faculty_publication_date[LANGUAGE_NONE][0]['value2']);
-
-      $date1 = date('Y-m-d', $date1);
-      $date2 = date('Y-m-d', $date2);
-      $arguments['q'][] = 'publicationDate:[' . $date1 . ' TO ' . $date2 . ']';
-    }
-
-    // Check for department id or name.
-    if (!empty($vars['bean']->field_faculty_pub_department)) {
-      $department = $vars['bean']->field_faculty_pub_department[LANGUAGE_NONE][0]['value'];
+    $nodePath = drupal_get_normal_path($_GET['q']);
+    $cid = $_GET['page'] ? $nodePath . "/" . $_GET['page'] : $nodePath;
```

Comment:
Not sure why this was reverted, but we do not want to cache the results of the experts database based on a path of the web express site - this current code would result in an editor not being able to see any changes they made to the query because the cache is based on a path in the web express site that will never change.

The key should be set to the experts query instead.